src/System.Windows.Forms/src/System/Windows/Forms/ListViewLabelEditAccessibleObject.cs

```diff
@@ -1,120 +1,64 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using Windows.Win32.System.Com;
 using Windows.Win32.UI.Accessibility;
-using static System.Windows.Forms.ListViewItem;
+using static System.Windows.Forms.ListViewItem.ListViewSubItem;
 using UiaCore = Interop.UiaCore;
 namespace System.Windows.Forms;
 
-internal unsafe class ListViewLabelEditAccessibleObject : AccessibleObject
+internal unsafe class ListViewLabelEditAccessibleObject : LabelEditAccessibleObject
 {
     private const string LIST_VIEW_LABEL_EDIT_AUTOMATION_ID = "1";
 
-    private readonly ListView _owningListView;
-    private readonly ListViewSubItem? _owningListViewSubItem;
-    private readonly ListViewItem? _owingListViewItem;
+    private readonly WeakReference<ListView> _owningListView;
     private readonly WeakReference<ListViewLabelEditNativeWindow> _labelEdit;
-    private readonly ListViewLabelEditUiaTextProvider _textProvider;
-    private int[]? _runtimeId;
+    private readonly LabelEditUiaTextProvider _textProvider;
 
-    public ListViewLabelEditAccessibleObject(ListView owningListView, ListViewLabelEditNativeWindow labelEdit)
+    public ListViewLabelEditAccessibleObject(ListView owningListView, ListViewLabelEditNativeWindow labelEdit) : base(owningListView, labelEdit)
     {
-        _owningListView = owningListView.OrThrowIfNull();
-        _owningListViewSubItem = owningListView._listViewSubItem;
-        _owingListViewItem = owningListView._selectedItem;
+        ArgumentNullException.ThrowIfNull(owningListView);
+
+        _owningListView = new(owningListView);
         _labelEdit = new(labelEdit);
         UseStdAccessibleObjects(labelEdit.Handle);
-        _textProvider = new ListViewLabelEditUiaTextProvider(owningListView, labelEdit, this);
+        _textProvider = new(owningListView, labelEdit, this);
     }
 
-    private protected override string AutomationId => LIST_VIEW_LABEL_EDIT_AUTOMATION_ID;
+    private ListViewSubItemAccessibleObject? owningSubItemAccessibleObject
+        => _owningListView.TryGetTarget(out ListView? target)
+        ? target._listViewSubItem is not null
+            ? target._listViewSubItem.AccessibilityObject as ListViewSubItemAccessibleObject
+            : null
+        : null;
```

Comment:
Believe this can be simplified to
```suggestion
        => _owningListView.TryGetTarget(out ListView? target)
        ? target._listViewSubItem?.AccessibilityObject as ListViewSubItemAccessibleObject
        : null;
```

Reply:
Done.